planning/behavior_path_planner/autoware_behavior_path_avoidance_by_lane_change_module/src/scene.cpp

```diff
@@ -273,16 +273,12 @@ double AvoidanceByLaneChange::calcMinAvoidanceLength(const ObjectData & nearest_
   return avoidance_helper_->getMinAvoidanceDistance(shift_length);
 }
 
-double AvoidanceByLaneChange::calcMinimumLaneChangeLength() const
+double AvoidanceByLaneChange::calc_minimum_dist_buffer() const
 {
-  const auto current_lanes = get_current_lanes();
-  if (current_lanes.empty()) {
-    RCLCPP_DEBUG(logger_, "no empty lanes");
-    return std::numeric_limits<double>::infinity();
```

Comment:
If I understand the code correctly, the new return value in case `current_lanes` is empty is `0` compared to `infinity()` previously. Is this okay ?
- the `calc_lc_length_and_dist_buffer()` returns `{}` when `lanes` is empty. 
- `{}` uses the [default constructor of `pair`](https://en.cppreference.com/w/cpp/utility/pair/pair) which uses the default constructor of `Boundary` which sets `min = 0.0`.

Reply:
@zulfaqar-azmi-t4 did you check this issue ?